content/blog/intro-to-gamedev/index.md

```diff
@@ -0,0 +1,560 @@
+
+## Tech requirements
+
+Keeping the scope in mind, let's try to deduce and compress what we need from a technology perspective:
+
+- We want the game to run on the browser, which pretty much inevitably means we are going to use either Javascript, or something that ends up turning into Javascript.
+- We have to do game-specific tasks such as render images and process input. Luckily the browser gives us good support for both of them through things like WebGL and Javascript's input system respectively.
+- We would also like it to be as responsive as possible since it makes for a better user experience.
+
+Note that we don't really need brazing fast graphic drawing for this since the game will not render thousands of objects at once, so using WebGL is probably overkill. However, it's a good way to get into the topic and so we'll take the chance to do it that way. WebGL is fairly complicated to work with from scratch, so after doing a bit of research I have decided we can use PixiJS to render stuff a pretty simple set of Javascript libraries for a 2D renderer that runs over WebGL (which in turn means we are using hardware acceleration to render stuff). We can also go a step further and try to use it with Typescript, to get that sweet, sweet type-checking.
+
+## Starting out
+
+In the world of web development, it's common
+We are going to use a starter template, which will solve some of these issues for us. Specifically, the starter is the following: https://github.com/llorenspujol/parcel-pixijs-quickstarter. It includes Parcel, PixiJS and TypeScript. PixiJS started shipping with Type information since version 5, which is the one being included in the starter template. This means we don't have to import extra packages to be able to type-check PixiJS code.
+
+Note that the only real requirement for this is Node and NPM.
+
+### Project setup
+
+Let's go ahead and clone the template repo:
+
+```
+git clone --depth 1 https://github.com/llorenspujol/parcel-pixijs-quickstarter.git DinoGame
+```
+
+Immediately, on our **DinoGame** folder we'll have a starting project app that combines TypeScript and PixiJS (and already includes the type information), bundled with Parcel which also provides some nifty features such as hot module reloading. At this point it's a good idea to run `npm install` to install all the package dependencies that are defined as standard node dependencies.
+
+We can actually go ahead and run the application by doing `npm run start`, which kicks off the build process and hosts the app on `https://localhost:1234/`. You'll see a cute animation of a character that looks like Bomberman.
+
+Let's review the code a little bit. The two TypeScript files that are important are the following: `main.ts`, on the root directory, and `src\app.ts`
```

Comment:
`src\app.ts` this looks like the only thing that uses `\`, don;t know if we really care about that tho